L1Trigger/Phase2L1ParticleFlow/plugins/L1MetPfProducer.cc

```diff
@@ -84,8 +136,68 @@ void L1MetPfProducer::produce(edm::StreamID, edm::Event& iEvent, const edm::Even
   iEvent.put(std::move(metCollection));
```

Comment:
I have another suggestion. Sorry that I missed it last time. 
The line
**std::unique_ptr<std::vector<l1t::EtSum>> metCollection(new std::vector<l1t::EtSum>(0));**
above may be changed to
**auto metCollection = std::make_unique<std::vector<l1t::EtSum>>(0);**
which is preferred (C++14 onwards).